Python/bytecodes.c

```diff
@@ -4993,7 +4993,7 @@ dummy_func(
             _Py_CODEUNIT *target = _PyFrame_GetBytecode(frame) + exit->target;
         #if defined(Py_DEBUG) && !defined(_Py_JIT)
             OPT_HIST(trace_uop_execution_counter, trace_run_length_hist);
-            if (lltrace >= 2) {
+            if (FT_ATOMIC_LOAD_UINT8_RELAXED(frame->lltrace) >= 2) {
```

Comment:
The frame cannot be access from another thread when on the stack, so the `FT_ATOMIC_LOAD_UINT8_RELAXED` shouldn't be needed.